### ISSUE TITLE
tabs don't load automatically when visiting them the first time (ff4b12pre)

### ISSUE THREAD

**Issue by lepster** [philikon/BarTab#104]
with firefox 4b12pre tabs are not loaded automatically when visiting them the first time. the just load if you hit refresh

**Comment by GJRobert** [comment]
Yes this happens on 4.0b13pre too :) Hope this can get back normal soon :) Thanks!
